### PR TITLE
Layout builders model builder

### PR DESCRIPTION
**Pull request by domsinclair**
Line 56 there is a n missing from person

### CODE REVIEW

File: docs/guide/layout-builders-model-builder-syntax.md [hunk 1]
```diff
@@ -4,28 +4,26 @@ title: 'DetailViewLayoutBuilders - ModelBuilder Syntax'
...
+As with [ModelBuilders](/guide/modelbuilders.md) this example will use  fluent syntax combined with the `WithDetailViewLayout`.
 
-> The overload `WithDetailViewLayout` takes an signature of `BuildLayoutFunctor` which is a method that just returns a `Layout` object. The second overload provides us handily a [`LayoutBuilder<Person>`](/guide/layout-builders-advanced-syntax.md). That way we don't need to create the builder our self.
+> The overload `WithDetailViewLayout` takes a signature of `BuildLayoutFunctor` which is a method that just returns a `Layout` object. The second overload provides a [`LayoutBuilder<T>`](/guide/layout-builders-advanced-syntax.md) obviating the need to create the builder independently.
 
 <<< @/guide/samples/layout-builders-model-builders/Module.cs{27,28}
 
-Because we use `ModelBuilders` to add the attribute to our `Person` class, we don't need to specify anything special:
+As  `ModelBuilders` are used to add the attribute tothe `Person` class, there is no requirement to specify anything special:
```
Comment (Paramethod) [inline docs/guide/layout-builders-model-builder-syntax.md:23]: tothe -> to the

File: docs/guide/layout-builders-model-builder-syntax.md [hunk 2]
```diff
@@ -4,28 +4,26 @@ title: 'DetailViewLayoutBuilders - ModelBuilder Syntax'
...
 
 <<< @/guide/samples/layout-builders-simple/Person.cs
 
 ::: tip
-This is the optimal way of adding Layouts to your classes you don't have source access to or want to keep your model clean from attributes.
-:::
\ No newline at end of file
+This is the optimal way to add Layouts to to classes to which source access unavailable or when it is desirable to keep the model clean from attributes.
```
Comment (Paramethod) [inline docs/guide/layout-builders-model-builder-syntax.md:28]: Repeated to to

File: docs/guide/layout-builders.md [hunk 3]
```diff
@@ -48,16 +51,16 @@ Usage in platform specific module is of course supported, but we will focus on t
 
 ## Usage
 
-To get you starting quickly there is simplified registration pattern, as well as a more advanced one. If you prefer to use [ModelBuilders](/guide/model-builders.md) there is also optimized integration provided.
+Xenial>Framework provides both a simple and advanced registration patter for LayoutBuilders as well as one specially optimsed for use within ModelBuilders.
```
Comment (Paramethod) [inline docs/guide/layout-builders.md:54]: patter -> pattern
optimsed -> optimized

File: docs/guide/layout-builders-record-syntax.md [hunk 4]
```diff
@@ -4,13 +4,13 @@ title: 'DetailViewLayoutBuilders - Record Syntax'
 
 # DetailViewLayoutBuilders - `Record` Syntax
 
-C#9 [introduced a new syntax for record's](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records#:~:text=C%23%209%20introduces%20records%2C%20a,types%20use%20value%2Dbased%20equality.). Because LayoutBuilders where implemented using this new feature, we are able to build layouts using the `with` expressions. They are not very different from initalizers, but they allow us to create a copy of a given record, which is really helpful to create a clean fluent syntax especially in combination with a **functional style** API.
+C#9 [introduced a new record syntax](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records#:~:text=C%23%209%20introduces%20records%2C%20a,types%20use%20value%2Dbased%20equality.) which has been implemented within Xenial>Framework LayoutBuilders ensuring that layouts can be built using `with` expressions. Although not very different from initalizers they make it possible to create a copy of a given record, which is particularly beneficial to a clean fluent syntax in combination with a **functional style** API.
```
Comment (biohazard999) [inline docs/guide/layout-builders-record-syntax.md:7]: Xenial>Framework -> `Xenial.Framework`

File: docs/guide/model-builders-managers.md [hunk 5]
```diff
@@ -4,24 +4,24 @@ title: ModelBuilders - BuilderManagers
 
 # ModelBuilders - BuilderManagers
 
-In our last examples we directly registered the managers in the `CustomizeTypesInfo` method. Because there could be a lot of `ModelBuilders` in a module, this becomes quickly a maintenance issue. That's why `Xenial.Framework` provides a solution for this problem: `BuilderManagers`.
+In the previous examples `ModelBuilders` were registered directly in the `CustomizeTypesInfo` method. As the number of ModelBuilders in an XAF module grows this has the potential to become a serious maintenace issue. To avoid this potential problem Xenial.Framework provides `BuilderManagers`. 
 
 ## XafBuilderManager
 
-We derive from the `XafBuilderManager` class and override the `GetBuilders` method to create our instance of the `DemoTaskModelBuilder` like we learned in the last chapter.
+To utilse `BuilderManagers` create a new class (by convention name it <ModuleName>BuilderManager) and derive it from the `XafBuilderManager` class. Within the class override the `GetBuilders` method to create an instance of the `DemoTaskModelBuilder`.
```
Comment (Paramethod) [inline docs/guide/model-builders-managers.md:11]: utilse -> utilize

**Pull request merged**